portal/_templates/footer-menu.html

```diff
@@ -8,21 +8,20 @@ <h1>{{ _("About") }}</h1>
             <li><a href="https://projectpythia.org/about.html">{{ _("About Project Pythia") }}</a></li>
             <li><a href="https://foundations.projectpythia.org/preamble/how-to-use.html">{{ _("How to use Pythia Foundations") }}</a></li>
             <li><a href="https://projectpythia.org/#the-project-pythia-team">{{ _("The Project Team") }}</a></li>
+            <li><a href="https://projectpythia.org/#how-to-cite">{{ _("How to Cite") }}</a></li>
           </ul>
         </div>
         <div class="col-9 col-sm-4 col-md-4 col-lg-3 footer-menu-col">
           <h1>{{ _("Give Feedback") }}</h1>
           <ul class="list-unstyled">
             <li><a href="mailto:projectpythia@ucar.edu">{{ _("Email us!") }}</a></li>
-            <li><a href="https://github.com/ProjectPythia/projectpythia.github.io/discussions">{{ _("Project Pythia Discussions") }}</a></li>
-            <li><a href="https://github.com/ProjectPythia/pythia-foundations/discussions">{{ _("Pythia Foundations Discussions") }}</a></li>
+            <li><a href="https://discourse.pangeo.io/c/education/project-pythia/60s">{{ _("Pangeo Discourse") }}</a></li>
           </ul>
         </div>
         <div class="col-9 col-sm-4 col-md-4 col-lg-3 footer-menu-col">
           <h1>{{ _("Contribute") }}</h1>
           <ul class="list-unstyled">
             <li><a href="https://projectpythia.org/contributing.html">{{ _("How to contribute to Project Pythia") }}</a></li>
-            <li><a href="https://foundations.projectpythia.org/appendix/how-to-contribute.html">{{ _("How to contribute to Pythia Foundations") }}</a></li>
```

Comment:
I added how-to-cite, removed the discussion pages that are inactive, added a link to the pangeo discourse, and removed the foundations specific contributors guide because I feel we would need to add the cookbook contributors guide here too and the general guide should link to both. 

I'm not sure on this point and am open to other opinions on whether to have ALL contributor's guides here.

Reply:
Once we're happy with the new footer, I'll repeat the process on foundations and the Cookbook gallery.